protoc-gen-gogo/generator/generator.go

```diff
@@ -609,10 +609,33 @@ type Generator struct {
 	indent           string
 	writeOutput      bool
 
-	customImports  []string
+	customImports  []*customImport
 	writtenImports map[string]bool // For de-duplicating written imports
 }
 
+// customImport represents a custom import in the generated package
+type customImport struct {
```

Comment:
Do we still need this?
I would like to keep the diff with golang/protobuf in generator.go to a minimum.